jekyll/_cci2/orbs-faq.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: classic-docs
+title: "Orbs FAQ"
+short-title: "Orbs FAQ"
+description: "FAQs for Orbs"
+categories: [configuring-jobs]
+order: 20
+---
+
+This document describes various questions and technical issues that you may find helpful when working with Orbs.
+
+* TOC
+{:toc}
+
+## Introduction
+{:.no_toc}
+
+When using Orbs or authoring your own Orbs, you may come across some issues that may not be directly addressed in the documentation. This section attempts to address specific issues and questions that have been relayed to CircleCI during the initial design and development phases with internal and external customers. Because Orbs development and design will be a continuous process, this section will be frequently updated as needed to ensure you the latest technical information is available.
+
+### Downloading, Integrating, and Testing Orbs
+
+* Question: How do I download, integrate and test Orbs?
+
+* Answer: You can invoke Orbs using the `orbs` stanza in version 2.1 or higher of a CircleCI configuration. For example, if you want to publish an Orb called  `hello-build` in the namespace `circleci` and have a published version `0.0.1`, invoke an orb like the example shown below:
+
+```orbs:
+hello: circleci/hello-build@0.0.1
```

Comment:
nit: indentation

Reply:
good catch, this is not a nit it is a bug!